test/plugins/test-trace-connect.js

```diff
@@ -190,6 +190,35 @@ describe('test-trace-connect', function() {
       });
     });
   });
+
+  it('should end spans even if client aborts', function(done) {
+    var app = connect();
+    app.use('/', function (req, res) {
+      setTimeout(function() {
+        res.end(common.serverRes);
+        setImmediate(function() {
+          var traces = common.getTraces(agent);
+          assert.strictEqual(traces.length, 1);
+          assert.strictEqual(traces[0].spans.length, 1);
+          var span = traces[0].spans[0];
+          common.assertSpanDurationCorrect(span, common.serverWait);
```

Comment:
Wouldn't we expect the span to have duration `common.serverWait / 2` since that is the time between start and abort?